nixos/modules/services/web-apps/lemmy.nix

```diff
@@ -61,12 +56,15 @@ in
           difficulty = mkOption {
             type = types.enum [ "easy" "medium" "hard" ];
             default = "medium";
-            description = "The difficultly of the captcha to solve.";
+            description = "The difficulty of the captcha to solve.";
           };
         };
 
-        options.database.createLocally = mkEnableOption "creation of database on the instance";
-
+        options.database.createLocalDatabase = mkOption {
+          type = types.bool;
+          default = true;
+          description = "Whether to create a local database automatically.";
+        };
```

Comment:
``` suggestion
        options.database.createLocalDatabase = mkEnableOption "automatic creation of a local database";
```

Reply:
Good Catch!

Reply:
This won't enable the option by default which is what we want.

Reply:
Why are you renaming this? The database word is redundant and many other NixOS modules use `database.createLocally`. Consistency and well established patterns are good.

Reply:
You have a point.